samples/sensor/vl53l0x/prj.conf

```diff
@@ -1,3 +1,4 @@
+CONFIG_STDOUT_CONSOLE=y
```

Comment:
This change should be in a separate patch, it has nothing to do with sensor units.

Reply:
It makes sure there're no inconsistencies in the repo, semantic inconsistencies in this case. Like, build won't fail if git bisect will fall inbetween 2 commits you propose, but there can be some erratic results still. E.g., tests could fail. But we don't have tests for that functionality, but if someone try to build the sample, they will get erratic results.

That said, I'm a big fan of separate commits, and will make this change, if only we had overall consistency with such split commits approach by different developers...